Cargo.toml

```diff
@@ -27,14 +27,18 @@ serde = {version = "1.0", optional = true}
 serde_json = {version = "1.0", optional = true}
 # rust-elgamal (via curve25519-dalek-ng) only works with digest 0.9, so pin this
 sha2 = "0.9"
-stderrlog = {version = "0.5", optional = true}
 structopt = {version = "0.3", optional = true}
 thiserror = "1.0"
+tracing = "0.1.35"
 tokio = { version = "1.19.2", optional = true, features = ["rt", "rt-multi-thread", "macros"] }
 axum = { version = "0.5.7", optional = true, features = ["http2"] }
 hyper = { version = "0.14.19", optional = true, features = ["client", "h2"] }
 hyper-tls = { version = "0.5.0", optional = true }
 axum-server = { version = "0.4.0", optional = true, features = ["rustls", "rustls-pemfile", "tls-rustls"] }
+tracing-subscriber = { version = "0.3.14", optional = true }
+tower-http = { version = "0.3.4", optional = true, features = ["trace"] }
```

Comment:
I think that we should continue to sort dependencies.  Since you added the out-of-order ones, maybe you can fix them now :)